plugins/LilyPad/Config.cpp

```diff
@@ -1393,8 +1394,13 @@ int BindCommand(Device *dev, unsigned int uid, unsigned int port, unsigned int s
     if (port > 1 || slot > 3 || padtype >= numPadTypes)
         return -1;
 
-    if (!sensitivity)
-        sensitivity = BASE_SENSITIVITY;
+    if (!sensitivity) {
+        if (((uid >> 16) & 0xFF) == ABSAXIS) {
+            sensitivity = 87183;
```

Comment:
87183 is the default sensitivity for the analog sticks in default.ini. But if you create a new binding for analog sticks you always got BASE_SENSITIVITY(which results in 65536) in previous LilyPad versions.